tests/unit/addon-exports-test.js

```diff
@@ -40,7 +40,8 @@ const HELPER_METHODS = [
 const EXTEND_METHODS = HELPER_METHODS.concat('registerExecutionContext');
 
 const EXPECTED_MACROS = [
-  'alias'
+  'alias',
+  'getter'
```

Comment:
I should have included this in #317. Ok to sneak it in here?

Reply:
No problem to include this in this PR :)